docs/Project-Configuration/Project-Commands.md

```diff
@@ -1424,9 +1454,15 @@ distribution. **Files uploaded with this command will also be viewable within th
 Parameters:
 
 -   `aws_key`: your AWS key (use expansions to keep this a secret).
--   `aws_secret`: your AWS secret (use expansions to keep this a secret).
--   `aws_session_token`: your temporary AWS session token (use expansions to keep this a secret). This cannot be used
-    with `visibility: signed`.
+-   `aws_secret`: your AWS secret (use expansions to keep this a secret)
+-   `aws_session_token`: your temporary AWS session token (use expansions to keep this a secret).
+    Note: If you are generating temporary credentials using `ec2.assume_role`, it is recommended
+    to pass in the role_arn directly to your s3 commands instead.
```

Comment:
Is the reason for this that `ec2.assume_role` credentials expire? If so, it may help to clarify that so people know that passing in the role protects against the expiration.

Reply:
I reworded it to be about what the user can do and what Evergreen does in response, let me know if it makes sense. The only reason to recommend it is shorter yamls, and prevents 1 AssumeRole call (Evergreen will ignore the temporary credentials passed in and will re-call AssumeRole with the assoicated role ARN if it found one)